lib/wibox/widget/textbox.lua

```diff
@@ -239,17 +262,13 @@ end
 -- @treturn table A new textbox widget
 -- @function wibox.widget.textbox
 local function new(text, ignore_markup)
-    local ret = base.make_widget()
+    local ret = base.make_widget(nil, nil, {enable_properties = true})
 
-    for k, v in pairs(textbox) do
-        if type(v) == "function" then
-            ret[k] = v
-        end
-    end
+    util.table.crush(ret, textbox, true)
 
-    ret._dpi = -1
-    ret._ctx = PangoCairo.font_map_get_default():create_context()
-    ret._layout = Pango.Layout.new(ret._ctx)
+    ret._dpi = -1 --FIXME this is unused, but .dpi is used but not defined
```

Comment:
This is supposed to be used above (see the other comment), but apparently I typo'd. Feel free to fix that.